python/py-pgspecial/Portfile

```diff
@@ -13,15 +14,15 @@ description         Meta-commands handler for Postgres Database
 long_description    This package provides an API to execute meta-commands \
                     (AKA “special”, or “backslash commands”) on PostgreSQL.
 
-python.versions     27 36 37 38
+python.versions     27 36 37 38 39
 
 homepage            https://pypi.python.org/pypi/${python.rootname}/
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
```

Comment:
also here you can remove `master_sites` and `distname` as these are the defaults